### DISCUSSION TITLE
Windows Download Link Broken

### DISCUSSION THREAD

**Discussion by Dartterious** [lllyasviel/Fooocus#1367] [General]
Does anyone have a working link to download this? The link on the main page give a 404 error.

**Comment by lllyasviel** [comment]
which link is broken?

**Comment by Dartterious** [comment]
This one https://github.com/lllyasviel/Fooocus/releases/download/release/Fooocus_win64_2-1-831.7

**Comment by Dartterious** [comment]
![image](https://github.com/lllyasviel/Fooocus/assets/109719059/937e525b-b442-44c4-a750-44b90ba5d47f)

**Reply by lllyasviel** [reply]
where you get this link? what browser you are using?
on normal browser it is 

https://github.com/lllyasviel/Fooocus/releases/download/release/Fooocus_win64_2-1-831.7z

**Reply by Dartterious** [reply]
I just clicked the link you provided on the main code page. Using Edge browser.

**Reply by Dartterious** [reply]
Weird. Your link above worked just fine. ¯\_(ツ)_/¯

Sorry for the inconvenience and thanks for your reply

**Reply by lllyasviel** [reply]
are you sure this is not user mistake since you lack a "z" in the end of link

**Reply by Dartterious** [reply]
No I'm just shit at copy/paste lol. But possibly user error - I think it had something to do with needing to be logged in and at the time I wasn't. Never had to log in before to download something off GitHub
